File: Java/BookStoreOOP/src/model/BookCategory.java
package model;

public class BookCategory {
    private String id;
    private String name;

    public BookCategory(String id, String name) {
        this.id = id;
        this.name = name;
    }

    //getter and setter
    public String getId() {
        return id;
    }
    public String getName() {
        return name;
    }
    public void setId(String id) {
        this.id = id;
    }

    public void setName(String name) {
        this.name= name;
    }
}

File: C#/BookStoreOOP/BookStoreOOP/Model/BookCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BookStoreOOP.Model
{
    public class BookCategory
    {
        private string id;
        private string name;

        public BookCategory(string id, string name)
        {
            this.id = id;
            this.name = name;
        }

        //getter and setter
        public string getId()
        {
            return id;
        }
        public string getName()
        {
            return name;
        }
        public void setId(string id)
        {
            this.id = id;
        }

        public void setName(string name)
        {
            this.name = name;
        }
    }
}
